doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,550 @@
+
+  * Fixed power-efficiency issues.
+  * Added experimental use of CryptoCell interrupt instead of busy-waits.
+* :ref:`lib_hw_cc310`:
+
+  * Added support for CryptoCell interrupt.
+
+nRF Bluetooth LE Controller
+===========================
+
+* Updated the :ref:`nrfxlib:ble_controller` libraries:
+
+  * Removed version numbers for the libraries.
+  * Added preliminary support for the S140 variant with the nRF5340 device.
+    The Bluetooth LE Controller for nRF5340 supports the same feature set as its nRF52 Series counterpart.
+  * Moved some APIs to :ref:`nrfxlib:mpsl`.
+    The library must now be linked together with MPSL.
+  * Made Data Length Extensions a configurable feature.
+  * Fixed an issue where an assert could occur when receiving a packet with a CRC error after performing a data length procedure on Coded PHY.
+
+  For details, see the :ref:`nrfxlib:ble_controller_changelog`.
+
+Multi-Protocol Service Layer (MPSL)
+===================================
+
+* Updated the :ref:`nrfxlib:mpsl` libraries:
+
+  * Removed version numbers for the libraries.
+  * Added a library version with preliminary support for the nRF5340 device.
+    The feature set is the same as in the MPSL library for nRF52.
+
+  For details, see the :ref:`nrfxlib:mpsl_changelog`.
+
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* Added the following sample:
+
+  * :ref:`peripheral_gatt_dm` - demonstrates how to use the :ref:`gatt_dm_readme`.
+
+* :ref:`nrf_bt_scan_readme`:
+
+  * Added an option to update the initial connection parameters.
+
+* :ref:`nus_c_readme`, :ref:`bas_c_readme`, and :ref:`dfu_smp_c_readme`:
+
+  * Fixed an issue where it was not possible to subscribe to the service notifications more than once.
+
+* Updated the :ref:`central_uart` sample to handle data packets that are longer than 212 bytes.
+  Enabled UART flow control to avoid data loss.
+
+* Enabled UART flow control in the :ref:`peripheral_uart` sample to avoid data loss.
+
+* Fixed the :ref:`ble_throughput` sample, which did not achieve the expected throughput measurements.
+
+
+NFC
+---
+
+* Added the following libraries:
+
+  * :ref:`nfc_t4t_cc_file_readme` - reads and parses the Capability Container file that can be found in the Type 4 Tag.
+  * :ref:`nfc_t4t_hl_procedure_readme` - performs the NDEF detection procedure for the Type 4 Tag.
+  * :ref:`tnep_tag_readme` - implements the Tag NDEF Exchange Protocol (TNEP) for a Tag device.
+  * :ref:`tnep_poller_readme` - implements the Tag NDEF Exchange Protocol (TNEP) for a Poller device.
+
+* Added the following samples:
+
+  * :ref:`nfc_tnep_tag` and :ref:`nfc_tnep_poller` -  demonstrate how to use the Tag NDEF Exchange Protocol (TNEP).
+  * :ref:`nrf-nfc-system-off-sample` - demonstrates how to wake the device from System OFF mode using NFC.
+    This sample was ported from the nRF5 SDK.
+
+* Extended the :ref:`nfc_tag_reader` sample with parsing and printing of the Type 4 Tag content.
+
```

Comment:
Moved the NFC Platform implementation to the _fw-nrfconnect-nrf_ repo (related docs in nrfxlib: `:ref:` nfc_integration_notes)